community/installation-guides/panel/centos8.md

```diff
@@ -28,15 +28,19 @@ systemctl start mariadb
 systemctl enable mariadb
 ```
 
-### PHP 7.2
-We recommend the remi repo to get the latest php packages.
+### PHP 7.3 
+We recommend the Remi modular repo to get the latest php packages.
 
 ```bash
 ## Get dnf updates
 dnf update -y
 
-## Install PHP 7.2
-dnf install -y php php-common php-fpm php-cli php-json php-mysqlnd php-gd php-mbstring php-pdo php-zip php-bcmath php-dom php-opcache
+## Install EPEL & Remi Modular Repository
+dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+dnf -y install https://rpms.remirepo.net/enterprise/remi-release-8.rpm
+
+## Install PHP 7.3
+dnf -y module install php:remi-7.3/common
```

Comment:
Yes, especially when the default AppStream only covers PHP 7.3 and not PHP 7.4 as of now. 

By using the remi repo, when we need to update to PHP 7.4, it would be quite simple - just reset the module, run dnf module install php:remi-7.4/common, and run distro-sync.

Reply:
On another note, the remi repo now also provides an appstream for redis 5 & 6, while the default repo only have redis 5.